test/LibraryManager.IntegrationTest/FileSaveRestoreTests.cs

```diff
@@ -0,0 +1,344 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using Microsoft.Test.Apex.VisualStudio;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    [TestClass]
+    public class FileSaveRestoreTests : VisualStudioLibmanHostTest
+    {
+        ProjectTestExtension _webProject;
+        ProjectItemTestExtension _libManConfig;
+        const string _projectName = @"TestProjectCore20";
+
+        [TestInitialize()]
+        public void initialize()
+        {
+            _webProject = Solution.ProjectsRecursive[_projectName];
+            _libManConfig = _webProject.Find(SolutionItemFind.FileName, "libman.json");
```

Comment:
>Find(SolutionItemFind.FileName, "libman.json") [](start = 40, length = 46)

This should be in the project root, so use `_webProject["libman.json"]`

Reply:
Will try

---
In reply to: [194548636](https://github.com/aspnet/LibraryManager/pull/113#discussion_r194548636) [](ancestors = 194548636)